CONDUCT.md

```diff
@@ -0,0 +1,111 @@
+# Scikit-Optimize Code of Conduct
+
+Scikit-Optimize is an engaged and respectful community made up of people
+from all over the world. Your involvement helps us to further our
+mission.
+
+The goal of scikit-optimize is to become the go to project for black box
+optimization, freely available to the reader and everyone else. As any other
+open source project, it requires a community of people who maintain and
+contribute to it. The quality of the project depends on the engagement and
+size of the community; thus you as a user of skopt directly benefit from
+creating positive environment that encourages people to contribute to the
+project.
+
+To ensure that we build the best possible environment for our community, we
+introduced this code of conduct. When these guidelines are not followed, it
+can be detrimental to the community, and in turn not beneficial to the users of
+scikit-optimize.
+
+Naturally, this implies diversity of ideas and perspectives on often complex
```

Comment:
Ah here there seem to be a break of the flow with the previous paragraph.
Maybe start here like "We receive contributions from people all over the world. Naturally, ...."